addons/sms/tools/sms_api.py

```diff
@@ -56,7 +56,7 @@ def _get_sms_api_error_messages(self):
         We prefer a dict instead of a message-per-error-state based method so that we only call
         config parameters getters once and avoid extra RPC calls."""
         buy_credits_url = self.env['iap.account'].sudo().get_credits_url(service_name='sms')
-        buy_credits = f'<a href="{buy_credits_url}" target="_blank">%s</a>' % _('Buy credits.')
+        buy_credits = f'<a href="{buy_credits_url}" target="_blank">{_("Buy credits.")}</a>'
```

Comment:
This makes the term untranslatable, the translation framework is not smart enough to look into fstring expressions. You need to either convert to all % formatting, or add a variable I guess :/